### PR TITLE
Card: implementation of Ads Card

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1438
[Problem] Ads Card: add widget imeplementation
[Solution]
 - added css styles
 - added examples
[Screenshot]
image

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/common/card.less [hunk 1]
```diff
@@ -1,151 +1,206 @@
...
+	.ui-header {
+		display: flex;
+		height: 46 * @px_base;
+		padding: 0 20 * @px_base;
+
+		.ui-title {
+			margin-top: 24 * @px_base;
+			margin-bottom: 3 * @px_base;
+			font-size: 15 * @sp_base;
+			color: #333333;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/card.less]: Don't you need a variable for color?

File: src/css/profile/mobile/common/card.less [hunk 2]
```diff
@@ -1,151 +1,206 @@
...
+		.ui-controls {
+			order: 2;
+			display: flex;
+			margin-top: 20 * @px_base;
+			height: 26 * @px_base;
+
+			.ui-btn {
+				width: 26 * @px_base;
+				height: 26 * @px_base;
+				background-color: transparent;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/card.less:42]: ditto. following colors~

File: src/css/profile/mobile/common/card.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/card.less]: You are right. I have to change this. Thanks.

File: examples/mobile/UIComponents/webclip/ads-card-3/css/style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.app-banner-1 {
```
Comment (singa2000) [inline examples/mobile/UIComponents/webclip/ads-card-3/css/style.css:1]: I just have a question.
If the official GUI guide is released, the "app-banner-1" will be included in the tau css library, right?

Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/webclip/ads-card-3/css/style.css:1]: No. This class is only for this card and reference only to this specific image. Just this photo looks better (in my opinion) when is little higher positioned. The developer can adjust some elements according to him needs, I think.

Comment (singa2000) [reply examples/mobile/UIComponents/webclip/ads-card-3/css/style.css:1]: Ok. I see. Thanks.

**Review by singa2000 — APPROVED** [review]
LGTM

**Pull request merged**